moveit_ros/moveit_servo/src/utils/command.cpp

```diff
@@ -250,8 +250,7 @@ JointDeltaResult jointDeltaFromPose(const PoseCommand& command, const moveit::co
     }
     if (!is_planning_frame)
     {
-      RCLCPP_WARN_STREAM(getLogger(),
-                         "Command frame is: " << command.frame_id << " expected: " << servo_params.planning_frame);
+      RCLCPP_WARN_STREAM(getLogger(), "Command frame is: " << command.frame_id << " expected: " << planning_frame);
```

Comment:
Isn't this an error? What is the resulting joint_position_delta in that case?

Reply:
Yeah, nothing happens in this case (delta is zero) so it probably should be an error.